visualization.shadertoy/resources/presets.json.md

```diff
@@ -0,0 +1,42 @@
+# JSON file for user defined shader lists
+
+The file `presets.json` is used to define custom shaders.
+A custom path to the shader files (GLSL) can be declared in the add-on settings.
+
+Contents of `presets.json` should be:
+1. Name of preset. Either hardcoded string or defined in `resources/language/.../strings.po`
+2. GLSL shader to use. if no custom path was declared, the default `resources/` will be used.
+   Set custom path to './' to use shaders from the same path as your `presets.json` file.
+3. Channel value 0. Defines the texture to use (e.g. PNG file). Can also be set to "audio"
+   to pass related stream data within texture. The PNG and GLSL files should be located
+   in the same directory.
+4. Channel value 1 definition.
+5. Channel value 2 definition.
+6. Channel value 3 definition.
+7. *[Optional]* append string `"gl_only"` to make this preset only available
+   to OpenGL.
+
+Example:
+-------------------
+~~~javascript
+{ "presets":[
+
+/* Default way used by the addon itself */
+[ "The Disco Tunnel by poljere",  "discotunnel.frag.glsl",      "tex02.png", "tex15.png", "audio", "" ],
+
+/* Description defined in strings.po */
+[ 30100,                          "audioeclipse.frag.glsl",     "audio", "", "", "" ],
+
+/* Define custom file paths */
+[ "My own one",                   "/path/to/glsl/my.frag.glsl", "audio",    "/path/to/pngs/my_image.png", "", "" ],
+
+/* Preset only available to OpenGL (see 7)  */
+[ "Polar Beats by sauj123",       "polarbeats.frag.glsl",       "audio", "", "", "", "gl_only" ],
+
+/* GLSL and PNG files are in the same folder as presets.json */
+[ "Another one",                  "./another.frag.glsl",        "./my_01.png", "./my_02.png", "audio", "" ],
+
+/* Select texture image by file name, use default `resources/` dir */
+[ "Fractal Land by Kali",         "fractalland.frag.glsl",      "tex02.png", "tex15.png", "audio", "" ]
+]}
+~~~
```

Comment:
Have it updated. For here and also strings.po are texts complete equal, only about the format are there differences and for this file much better human readable :sweat_smile: .

Only has leaved the additional line break from strings.po away, as there then nice visible in single page view:
![Bildschirmfoto von 2021-09-10 22-12-54](https://user-images.githubusercontent.com/6879739/132912363-a6a6a42f-5813-4b2b-980a-ff1a8bcb651d.png).

Further has leaved the channel numbers by begin of 0, also not likes to have 0 as first, only use ShaderToy it the same, e.g. https://www.shadertoy.com/view/4ljGD1 and also on other settings already with begin as 0.

Thanks a lot @howie-f :+1: